packages/lesswrong/server/resolvers/ultraFeedResolver.ts

```diff
@@ -64,85 +64,54 @@ export const ultraFeedGraphQLTypeDefs = gql`
   }
 `
 
+// items now carry `type`
+type SampledItem =
+  | { type: "feedCommentThread"; feedCommentThread: FeedCommentsThread }
+  | { type: "feedPostWithContents"; feedPost: FeedFullPost }
+  | { type: "feedPost"; feedPostStub: FeedPostStub }          // ← NEW stub
```

Comment:
what does that comment mean?  (and why is this a stub now?)

Reply:
For bookmarks I have the repo return the ids but not post contents (unlike recombee and latest posts), so there are now posts that have contents and post that don't...Shoggoth labelled the later as stubs, I'm not crazy about the name